pages/auth/login.vue

```diff
@@ -149,9 +150,14 @@ export default {
       return this.err;
     },
 
+    errorToDisplay() {
+      return this.customLoginError?.showMessage && this.customLoginError?.message && this.errorMessage ? `${ this.customLoginError.message } \n ${ this.errorMessage }` : this.errorMessage ? this.errorMessage : '';
```

Comment:
I found this long ternary statement hard to read. To me it would be easier to understand if it was something like this:

```js
if (this.customLoginError?.showMessage && this.customLoginError?.message && this.errorMessage) {
  return `${ this.customLoginError.message } \n ${ this.errorMessage }`;
}

if (this.errorMessage) {
  return this.errorMessage;
}

return '';
```